sdk/storage/inc/blobs/blob_responses.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "protocol/blob_rest_client.hpp"
+
+#include <limits>
+#include <string>
+#include <utility>
+
+namespace Azure { namespace Storage { namespace Blobs {
+
+  struct BlobDownloadInfo
+  {
+    std::string ETag;
+    std::string LastModified;
+    int64_t ContentLength = 0;
+    BlobHttpHeaders HttpHeaders;
+    std::map<std::string, std::string> Metadata;
+    Blobs::BlobType BlobType = Blobs::BlobType::Unknown;
+    Azure::Core::Nullable<bool> ServerEncrypted;
```

Comment:
Why is it `Nullable`?

Reply:
I cannot remember. But I changed it from non-nullable to nullable. There must be an API that doesn't return this value in response headers.

Reply:
Maybe `GetBlob` REST with content range does not return this. Please give it a try when you have time. Not priority.